ruma-events/src/room/message.rs

```diff
@@ -59,6 +65,57 @@ pub enum MessageEventContent {
     Video(VideoMessageEventContent),
 }
 
+/// Enum modeling the different ways relationships can be expressed in a
+/// `m.relates_to` field of an m.room.message event.
+#[derive(Clone, Debug, Deserialize, Serialize)]
+#[serde(from = "RelatesToJsonRepr")]
+#[serde(into = "RelatesToJsonRepr")]
+pub enum Relation {
+    /// A reference to another event.
+    Reference(Reference),
+    /// An annotation to an event.
+    Annotation(Annotation),
```

Comment:
I think we can remove this variant here, right?

Reply:
Well who knows, perhaps there will be an annotation that isn't an `m.reaction`. For now it's only used for reactions but the naming of it seems to suggest that it's meant to be a more general thing.

Reply:
Okay, if you think it's a good idea to have it here as well, I'm fine with that. Just feature-gate it along with the other ones please.